### PR TITLE
[refact] core: change private function name to Symbol from being called outside.

### PR DESCRIPTION
**Pull request by iyuq**
##### Checklist

- [x] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

File: lib/egg.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ const ISCLOSE = Symbol('EggCore#isClose');
 const CLOSE_PROMISE = Symbol('EggCore#closePromise');
 const ROUTER = Symbol('EggCore#router');
 const EGG_LOADER = Symbol.for('egg#loader');
+const INIT_READY = Symbol('EggCore#_initReady');
```
Comment (popomore) [inline lib/egg.js]: _initReady > initReady

Comment (iyuq) [inline lib/egg.js]: 没毛病

Comment (iyuq) [inline lib/egg.js]: 已修改

**Comment by popomore** [comment]
下次注意 commit message https://eggjs.org/zh-cn/contributing.html#commit-提交规范

**Comment by iyuq** [comment]
好的。

**Comment by fengmk2** [comment]
@iyuq 👍

**Pull request merged**